gw_eccentricity/eccDefinition.py

```diff
@@ -156,12 +158,42 @@ def __init__(self, dataDict, num_orbits_to_exclude_before_merger=2,
                 Default: 2.
 
         precessing: bool, default=False
-            Whether the system is precessing or not. For precessing systems,
-            the `dataDict` should contain modes in the coprecessing frame. For
-            nonprecessing systems, there is no distiction between the inertial
-            and coprecessing frame since they are the same.
-
-            Default is False which implies the system to be nonprecessing.
+            Indicates whether the system is precessing. For precessing systems,
+            the (2, 2) and (2, -2) modes in the coprecessing frame are required
+            to compute `amp_gw`, `phase_gw`, and `omega_gw` (see
+            `get_amp_phase_omega_gw`), which are used to determine eccentricity.
+
+            For precessing systems, waveform modes in the coprecessing frame
+            must be provided. This can be done in two ways:
+                - Set `frame="coprecessing"` and supply the coprecessing modes
+                  directly via `dataDict`.
+                - Set `frame="inertial"` and provide the inertial frame modes
+                  via `dataDict`. In this case, the modes in `dataDict` are
+                  rotated internally (see `rotate_modes`) before further
+                  computation. To get the coprecessing modes from the inertial
+                  modes accurately, `dataDict` must include all modes for
+                  `ell=2`, i.e., (2, -2), (2, -1), (2, 0), (2, 1) and (2, 2).
+
+            For nonprecessing systems, the inertial and coprecessing frames are
+            equivalent, so there is no distinction.
+
+            Default is `False`, indicating the system is nonprecessing.
```

Comment:
For clarity, say that for nonprecessing, it's sufficient to include only the (2,2) mode.

Reply:
Done